src/main/java/filemanager/interceptor/RequestKeyCheckInterceptor.java

```diff
@@ -0,0 +1,41 @@
+package filemanager.interceptor;
+
+import filemanager.exceptions.DateIsEmptyException;
+import filemanager.model.Command;
+import org.aopalliance.intercept.MethodInterceptor;
+import org.aopalliance.intercept.MethodInvocation;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.ForbiddenException;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class RequestKeyCheckInterceptor implements MethodInterceptor {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(RequestKeyCheckInterceptor.class);
+
+    @Override
+    public Object invoke(MethodInvocation methodInvocation) throws Throwable {
+        Object[] methodArguments = methodInvocation.getArguments();
+        String key = (String) methodArguments[0];
+        Command command = (Command) methodArguments[1];
+        if (key.isEmpty()) {
+            throw new ForbiddenException();
+        } else {
+            if (command.getDate() == null) {
+                throw new DateIsEmptyException("The date field is required!");
```

Comment:
How do you know here that date is required? That should be up to Resource to decide. So I'd prefer to just remove this check for now.